src/ol_openedx_canvas_integration/tasks.py

```diff
@@ -3,7 +3,7 @@
 import logging
 from functools import partial
 
-from celery import task
+from celery import shared_task  # pylint: disable=import-error
```

Comment:
We're not using pylint in this repository so you can remove this comment. Instead we are relying on flake8 and the `wemake-python-styleguide` collection of plugins.

Reply:
@blarghmatey Thanks for identifying that. Removed the pylint comment.